runtime/laos/src/benchmarks.rs

```diff
@@ -21,7 +21,7 @@ frame_benchmarking::define_benchmarks!(
 	[pallet_utility, Utility]
 	[pallet_multisig, Multisig]
 	[pallet_proxy, Proxy]
-	[pallet_balances, Balances] // We use default weights from polkadot-sdk as benchmarking functions does not support existential deposit zero [pallet_vesting, Vesting]
+	// [pallet_balances, Balances] // We use default weights from polkadot-sdk as benchmarking functions does not support existential deposit zero [pallet_vesting, Vesting]
```

Comment:
I don't understand why is this being commented out. Is it because we don't need to benchmark it if we use the default weights, if so wouldn't it be better to delete it instead of commenting it?